code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -248,28 +248,6 @@
 		return FALSE
 	return TRUE
 
-/obj/item/robot_module/standard
```

Comment:
could you make them admin-only instead of fully removing them, like secborgs?

wait a minute, don't we have a config option to disable standard borgs?

Reply:
```
	if(!CONFIG_GET(flag/disable_peaceborg))
		modulelist["Peacekeeper"] = /obj/item/robot_module/peacekeeper
	if(!CONFIG_GET(flag/disable_secborg))
		modulelist["Security"] = /obj/item/robot_module/security
```

Oh, we have a config option to disable secborgs and a config option to disable PK borgs, but no config option to disable standard borgs yet.

Maybe you could add one?

Reply:
We're not having a config option for this when nobody is going to maintain it.

Reply:
What exactly needs to be maintained?  I'll maintain it if necessary.